OpenRA.Mods.Common/Commands/ChatCommands.cs

```diff
@@ -48,7 +48,7 @@ public bool OnChat(string playername, string message)
 
 		public void RegisterCommand(string name, IChatCommand command)
 		{
-			Commands.Add(name.ToLowerInvariant(), command);
+			Commands[name.ToLowerInvariant()] = command;
```

Comment:
Maybe worth a comment explicitly stating that we choose to ignore earlier duplicates instead of crashing?